### ISSUE TITLE
Hangs when using a log scale

### ISSUE THREAD

**Issue by JP-Ellis** [bheisler/cargo-criterion#33]
It appears that `cargo-criterion` does not like it when I change the axis scaling of a group to be logarithmic.  For example, the code below will run through the tests, and at the end the `cargo-criterion` seems to hang indefinitely when the `plot_config` line is uncommented:

```rust
pub fn insert_empty(c: &mut Criterion) {
    let mut group = c.benchmark_group("BTreeMap Insert Empty");
    // group.plot_config(PlotConfiguration::default().summary_scale(AxisScale::Logarithmic));

    for i in [1, 10, 100, 1000, 10_000, 100_000] {
        group.bench_function(BenchmarkId::from_parameter(i), |b| {
            let mut sl: BTreeMap<usize, usize> = BTreeMap::new();
            let mut rng = StdRng::seed_from_u64(0x1234abcd);

            b.iter(|| {
                for _ in 0..i {
                    sl.insert(rng.gen(), rng.gen());
                }
            })
        });
    }
}
```

I haven't checked yet whether this is specific to the logarithmic scaling or other plot configuration options.

**Comment by Lemmih** [comment]
This has been fixed in `criterion-0.3.5` but hasn't percolated to `cargo-criterion` yet.

**Comment by Lemmih** [comment]
The fix has landed and will hopefully be released in `cargo-criterion-1.1.0`. Thank you for your report. I will close this issue once the new version has been released.

**Comment by bheisler** [comment]
Thanks for the report! I've published version 1.1.0.
